featuretools/entityset/entity.py

```diff
@@ -414,6 +414,9 @@ def delete_variables(self, variable_ids):
         Remove variables from entity's dataframe and from
         self.variables
         """
+        # check variable is a list
+        if isinstance(variable_ids, str):
```

Comment:
Maybe we should raise this error if the provided input is not a list. Otherwise, it wouldn't get raised if an integer or float was passed, for example. So, let's change the condition to `if not isinstance(variable_ids, list)` instead.